modules/tomcat/manifests/proxy.pp

```diff
@@ -5,30 +5,32 @@
 class tomcat::proxy 
 (
   # config variables for mod_proxy
-  $host_name            = $tomcat::params::host_name,
-#  $proxyfdir            = $tomcat::params::proxyfdir,
-  $proxy_conf_file      = $tomcat::params::proxy_conf_file,
+#  $dns_name            = $tomcat::params::dns_name,
+  $dns_name,
 )
 {
+  # config variables for mod_proxy
+  $proxy_conf_file      = "/etc/httpd/conf.d/tomcat.conf"
```

Comment:
Also you can manage httpd from tomcat module.
You need to have defined type in httpd module which you will call in  profiles::webapp::tomcat profiles.